spec/requests/api/submissions/update_spec.rb

```diff
@@ -125,17 +125,19 @@
         expect(response.status).to eq 201
         expect(@submission.reload.receipt_sent_at).to_not be_nil
         emails = ActionMailer::Base.deliveries
-        expect(emails.length).to eq 3
+        expect(emails.length).to eq 1
         admin_email = emails.detect { |e| e.to.include?("lucille@bluth.com") }
-        admin_copy = "We have received the following submission from: Jon"
-        expect(admin_email.html_part.body.to_s).to include(admin_copy)
-        expect(admin_email.text_part.body.to_s).to include(admin_copy)
+        expect(admin_email).to be_nil
+        # admin_copy = "We have received the following submission from: Jon"
+        # expect(admin_email.html_part.body.to_s).to include(admin_copy)
+        # expect(admin_email.text_part.body.to_s).to include(admin_copy)
 
         user_email = emails.detect { |e| e.to.include?("michael@bluth.com") }
-        user_copy =
-          "to evaluate whether we currently have a suitable market for it"
-        expect(user_email.html_part.body.to_s).to include(user_copy)
-        expect(user_email.text_part.body.to_s).to include(user_copy)
+        pp ["ROOP", user_email]
```

Comment:
Looks like a stray print line here ☝🏼

Reply:
Weird, I don't see that locally, maybe I got too clever with my rebases 😅 I'll sort that out and merge…